pyrokinetics/kinetics/KineticsReaderpFile.py

```diff
@@ -63,19 +64,30 @@ class KineticsReaderpFile(KineticsReader):
     def read(
         self,
         filename: PathLike,
-        second_filename: PathLike,
+        eq_file: Optional[PathLike] = None,
         time_index: int = -1,
-        time: float = None,
+        time: Optional[float] = None,
     ) -> Dict[str, Species]:
         """
         Reads in Osborne pFile. Your pFile should just be called, pFile.
 
-        Also reads a geqdsk file, which is in the same directory as the pFile, and assumed to be called geqdsk.
-
-        geqdsk file passed through second_filename.
+        Also reads a geqdsk file via eq_file, which is in the same directory as the
+        pFile, and assumed to be called geqdsk.
         """
-        # Read pFile, get generic data.
+        # eq_file must be provided
+        # Note: The 'Optional[T]' type hint just means the arg can be of type T or it
```

Comment:
It'd work just fine, but a static type checker might complain, as the default value `None` isn't of type `PathLike`. The comment is just pointing out that `Optional[T]` is really an alias for `Union[T, None]` (or maybe `Union[T, type(None)]`?), and so despite the implications of the name 'optional', in this case we do actually require the user to provide something.

In other words, it's me being weirdly pedantic about type hints, so it's probably best to just remove the type hint and the comment since they're actually degrading readability in this case, and that kinda defeats the whole point.